app/helpers/countries_helper.rb

```diff
@@ -0,0 +1,22 @@
+module CountriesHelper
+  def is_malaysia?
+    @country && @country.iso_3 == "MYS"
+  end
+
+  def is_japan?
```

Comment:
@Levia I spotted this `is_japan` method and was just curious about that, since everything else seems to be focused on Malaysia. Otherwise I am happy to merge!